api/internal/localizer/localizer.go

```diff
@@ -372,6 +423,78 @@ func (lc *localizer) localizeDir(path string) (string, error) {
 	return locPath, nil
 }
 
+// copyChartHome copies helm chart home path, relative to root that lc is at, to
+// the same location relative to dst and returns said relative copied location.
+//
+// If path indicates the default chart home, copyChartHome copies chart home to
+// the same location in dst, without following symlinks.
+func (lc *localizer) copyChartHome(path string) (string, error) {
+	if filepath.IsAbs(path) {
+		return "", errors.Errorf("absolute path %q not handled in alpha", path)
+	}
+	unclean := lc.root.Join(path)
+	// chart home may serve as untar destination
+	if !lc.fSys.Exists(unclean) {
+		return path, nil
+	}
+	// perform localize directory checks
+	ldr, err := lc.ldr.New(path)
+	if err != nil {
+		return "", errors.Wrap(err)
+	}
+	clean, err := filesys.ConfirmDir(lc.fSys, ldr.Root())
+	if err != nil {
+		log.Panicf("unable to establish validated directory %q: %s", path, err)
+	}
+	var copiedPath string
+	if unclean == lc.root.Join(defaultChartHome) {
+		copiedPath = defaultChartHome
+	} else {
+		copiedPath, err = filepath.Rel(lc.root.String(), clean.String())
+		if err != nil {
+			log.Panicf("no relative path between scoped directories %q and %q: %s", lc.root, clean, err)
+		}
+	}
+	err = lc.copyDir(clean, filepath.Join(lc.dst, copiedPath))
+	if err != nil {
+		return "", errors.WrapPrefixf(err, "unable to copy chart home %q", path)
+	}
+	return copiedPath, nil
+}
+
+// copyDir copies src to dst. copyDir does not follow symlinks.
+func (lc *localizer) copyDir(src filesys.ConfirmedDir, dst string) error {
+	err := lc.fSys.Walk(src.String(),
+		func(path string, info fs.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+			pathToCreate, err := filepath.Rel(src.String(), path)
+			if err != nil {
+				return errors.Wrap(err)
+			}
+			pathInDst := filepath.Join(dst, pathToCreate)
+			if info.Mode()&os.ModeSymlink == os.ModeSymlink {
```

Comment:
@natasha41575 As requested, I don't follow symlinks. 

As discussed before, our other option is to follow all symlinks that don't leave `scope` and write the file contents to the nominal (uncleaned) file paths in the chart home directory.